src/main/java/Duke.java

```diff
@@ -1,10 +1,37 @@
+import java.util.Objects;
+import java.util.Scanner;
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner reader = new Scanner(System.in);
+        Task[] items = new Task[100];
+        int counter = 0;
+        System.out.println("Hi, I'm bob");
+        System.out.println("What's up");
```

Comment:
Should this be extracted out as a print welcome message method to avoid long methods?

Reply:
Thank you for all your encouragement and insightful advice! I will be making these changes accordingly 👍